assets/elm/Terms/Models.elm

```diff
@@ -24,39 +38,63 @@ initTerms =
     Terms "Unknown" [] []
 
 
-rankFields : List String
+combinedHeaders : List Workflow
+combinedHeaders =
+    List.map Combined <| coreFields ++ rankFields
+
+
+sciNameHeaders : List Workflow
+sciNameHeaders =
+    List.map WholeName <| coreFields ++ sciNameFields
+
+
+splitNameHeaders : List Workflow
+splitNameHeaders =
+    List.map SplitName <| coreFields ++ rankFields
+
+
+
+-- PRIVATE
+
+
+rankFields : List Term
 rankFields =
-    [ "kingdom"
-    , "subKingdom"
-    , "phylum"
-    , "subPhylum"
-    , "superClass"
-    , "class"
-    , "subClass"
-    , "cohort"
-    , "superOrder"
-    , "order"
-    , "subOrder"
-    , "infraOrder"
-    , "superFamily"
-    , "family"
-    , "subFamily"
-    , "tribe"
-    , "subTribe"
-    , "genus"
-    , "subGenus"
-    , "section"
-    , "species"
-    , "subSpecies"
-    , "variety"
-    , "form"
+    [ Term "kingdom" True "something" "http://example.org"
+    , Term "subKingdom" True "something" "http://example.org"
+    , Term "phylum" True "something" "http://example.org"
+    , Term "subPhylum" True "something" "http://example.org"
+    , Term "superClass" True "something" "http://example.org"
+    , Term "class" True "something" "http://example.org"
+    , Term "subClass" True "something" "http://example.org"
+    , Term "cohort" True "something" "http://example.org"
+    , Term "superOrder" True "something" "http://example.org"
+    , Term "order" True "something" "http://example.org"
+    , Term "subOrder" True "something" "http://example.org"
+    , Term "infraOrder" True "something" "http://example.org"
+    , Term "superFamily" True "something" "http://example.org"
+    , Term "family" True "something" "http://example.org"
+    , Term "subFamily" True "something" "http://example.org"
+    , Term "tribe" True "something" "http://example.org"
+    , Term "subTribe" True "something" "http://example.org"
+    , Term "genus" True "something" "http://example.org"
+    , Term "subGenus" True "something" "http://example.org"
+    , Term "section" True "something" "http://example.org"
+    , Term "species" True "something" "http://example.org"
+    , Term "subSpecies" False "something" "http://example.org"
+    , Term "variety" True "something" "http://example.org"
+    , Term "form" True "something" "http://example.org"
     ]
 
 
-coreFields : List String
+coreFields : List Term
 coreFields =
-    [ "taxonId", "taxonrank", "scientificName", "scientificNameAuthorship" ]
+    [ Term "taxonId" True "something" "http://example.org"
+    , Term "taxonrank" True "something" "http://example.org"
+    ]
 
 
-allFields =
-    coreFields ++ rankFields
+sciNameFields : List Term
+sciNameFields =
```

Comment:
Similar to above, thoughts on favoring the full spelling instead of the abbreviation?